app/templates/components/wikia-ui-components/wikia-card-crosswiki-list.hbs

```diff
@@ -11,7 +11,11 @@
     </li>
     {{#if includePostsForQuery}}
       {{#if (equal key 1)}}
-        {{post-search-interrupt query=includePostsForQuery isCrossWiki=true}}
+        {{post-search-interrupt
+          query=includePostsForQuery
```

Comment:
Out of curiosity is there any linting rule or standard when to do this?

Reply:
I don't think there's a linting rule, it just felt like the right thing to do. The line seemed too long when I added the additional attribute.